pom.xml

```diff
@@ -112,7 +117,8 @@
                         <image>416670754337.dkr.ecr.eu-west-2.amazonaws.com/ci-corretto-build-21:latest</image>
                     </from>
                     <to>
-                        <image>416670754337.dkr.ecr.eu-west-2.amazonaws.com/local/limited-partnerships-api:latest</image>
+                        <image>416670754337.dkr.ecr.eu-west-2.amazonaws.com/local/limited-partnerships-api:latest
+                        </image>
```

Comment:
I don't think this needed to change - image tag can remain on the same line (120).

Reply:
Is that manual formatting or auto-formatting from Google Java formatter? I would say revert this.

Reply:
But why is a Java formatter changing XML files?

Reply:
I don't see why it is a problem